### PR TITLE
MGMT-5240 - Add recently released 4.8.0-fc.0 to the default versions JSON

### PR DESCRIPTION
**Pull request by omertuc**
This will make it so that 4.8.0-fc.0 is available in staging/production
environments as a beta, non-default version.

Also updated the `hack/get_ocp_versions_for_testing.sh` script so it no
longer appends the "fake" 4.8 image for single-node.

However the override code is still kept around behind this gate to provide
an example of how we can do a similar override in the future.

### CODE REVIEW

File: hack/get_ocp_versions_for_testing.sh [hunk 1]
```diff
@@ -4,7 +4,18 @@ set -euo pipefail
...
-# Append single node image as 4.8 for local deployments
-"${PROJECT_ROOT_DIR}"/tools/handle_ocp_versions.py --src ./default_ocp_versions.json --ocp-override ${OPENSHIFT_SNO_IMAGE} --name-override single-node-4.8-alpha --version-override 4.8 --skip-verify | tr -d "\n\t "
+# This is currently disabled because the 4.8-fc image has been released and we already
+# include it in `./default_ocp_versions.json`.
+# The code is still kept around behind this gate to provide an example of how we can do
+# a similar override in the future.
+OVERRIDE_4_8_WITH_SINGLE_NODE=0
+
+if [[ ${OVERRIDE_4_8_WITH_SINGLE_NODE} == 1 ]]; then
+    OPENSHIFT_SNO_IMAGE="quay.io/openshift-release-dev/ocp-release-nightly:4.8.0-0.nightly-2021-03-16-221720"
```
Comment (gamli75) [inline hack/get_ocp_versions_for_testing.sh:15]: why you are using the fc version here?

**Review by gamli75 — COMMENTED** [review]
Did you test this version to verify that the cluster installation was finished successfully?

**Comment by omertuc** [comment]
> Did you test this version to verify that the cluster installation was finished successfully?

Not yet. Will do

/hold

File: hack/get_ocp_versions_for_testing.sh (hunk 1 above)
Comment (omertuc) [reply hack/get_ocp_versions_for_testing.sh:15]: this is now just meaningless old code used to serve as an example of how to do an override, it doesn't run, and that value doesn't matter

**Comment by omertuc** [comment]
Forced push due to conflicts from https://github.com/openshift/assisted-service/pull/1517

**Comment by omertuc** [comment]
/unhold

Installed successfuly

@gamli75

**Comment by eranco74** [comment]
LGTM

**Comment by eranco74** [comment]
> /hold Do we want this in production right now?
> 
> @gamli75 @romfreiman @eranco74

/unhold
We want it in prod in the next release (2 weeks)

**Pull request merged**